suzieq/cli/sqcmds/NetworkCmd.py

```diff
@@ -59,7 +59,7 @@ def show(self, os: str = "", vendor: str = "", model: str = "",
         return self._gen_output(df)
 
     @command("find", help="find where an IP/MAC address is attached")
-    @argument("address", type=str, description="IP/MAC address to find")
+    @argument("address", type=str, description="IP/MAC addresses, in quotes, space separated")
```

Comment:
Please just split this line in 2 because the linter is complaining
```suggestion
    @argument("address", type=str,
              description="IP/MAC addresses, in quotes, space separated")
```

Reply:
Fixed. I saw the linter fail.